web/src/components/molecules/Integration/IntegrationHeader/index.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from "@emotion/styled";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import PageHeader from "@reearth-cms/components/atoms/PageHeader";
+import { useT } from "@reearth-cms/i18n";
+
+type Props = {
+  handleConnect: () => void;
```

Comment:
functions in props should be `onSomeEvent`. `handleSomeEvent` should be locally declared

Reply:
@KaWaite , fixed.